src/bun.js/bindings/webcore/JSEventEmitter.cpp

```diff
@@ -224,7 +225,7 @@ void JSEventEmitter::finishCreation(VM& vm)
 
 JSObject* JSEventEmitter::createPrototype(VM& vm, JSDOMGlobalObject& globalObject)
 {
-    auto* structure = JSEventEmitterPrototype::createStructure(vm, &globalObject, globalObject.objectPrototype());
+    auto* structure = JSEventEmitterPrototype::createStructure(vm, &globalObject, JSEventTarget::prototype(vm, globalObject));
```

Comment:
this is definitely not correct

Reply:
The goal here was to inherit the EventTarget prototype - should this actually exist on JSEventEmitter directly?

Reply:
E.g. addEventListener, removeEventListener, dispatchEvent, etc
![CleanShot 2025-05-21 at 18 47 21@2x](https://github.com/user-attachments/assets/9dc88d6f-44c4-48db-b9e2-5f3faee36b9e)